team-config.yml

```
@@ -689,11 +689,19 @@ et:
     build_notices_channel: "#et-builds"
   tags:
     application: employment-tribunals
-ds:
+fis-ds-web:
```

the key is the product, you should not have multiple apps under here.

use `fis`

```suggestion
fis:
```